### PR TITLE
Add check for min/max of votes totals.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: tests/system/action/poll/test_vote.py [hunk 1]
```diff
@@ -165,6 +169,47 @@ def test_vote_correct_pollmethod_YN(self) -> None:
...
+                "user_id": 1,
+                "value": {"11": "Y", "12": "N"},
+            },
+        )
+        self.assert_status_code(response, 400)
+        assert (
+            "Total amount of votes is not in min-max-interval."
+            in response.data.decode()
+        )
+        self.assert_model_not_exists("vote/1")
```
Comment (FinnStutzenstein) [inline tests/system/action/poll/test_vote.py:211]: Please also test the less case (e.g. set min and max to 2 and give 2 vote).

**Review by FinnStutzenstein — COMMENTED** [review]
Looks good!

File: tests/system/action/poll/test_vote.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/poll/test_vote.py:211]: Add a test to the less case, set min and max to 2 and give a 1 vote.

Comment (FinnStutzenstein) [reply tests/system/action/poll/test_vote.py:211]: Ok

**Review by FinnStutzenstein — APPROVED** [review]

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**